torch/nn/modules/flatten.py

```diff
@@ -10,8 +10,8 @@ class Flatten(Module):
     Flattens a contiguous range of dims into a tensor. For use with :class:`~nn.Sequential`.
 
     Shape:
-        - Input: :math:`(N, *dims)`
-        - Output: :math:`(N, \prod *dims)` (for the default case).
+        - Input: :math:`(*, S_{start}, *, S_{end}, *)`
+        - Output: :math:`(*, \prod_{i=start}^{end} S_{i}, *)` (for the default case).
```

Comment:
nit: I don't think "(for the default case)" is needed now since the more precise edit should cover all cases I believe